### PR TITLE
Update Files.fr-FR.xlf

### PR DESCRIPTION
**Pull request by Gurgarath**
Added multiple missing entries in fr-FR
Warning: Disque Cloud can be renamed to "Disque nuagique" but this is most-likely used in fr-CA. For logical purposes, "Disques Cloud" have been used as it is widely used in the sheet already.

**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #issue...
- Related #issue...

**Details of Changes**
Add details of changes here.
- Added...
- Fixed...

**Validation**
How did you test these changes?
- [ ] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

**Comment by yaichenbaum** [comment]
@Gurgarath can you take a look at #6007?

**Comment by Gurgarath** [comment]
> 
> 
> @Gurgarath can you take a look at #6007?

This is a solid translation that doesn't collide with mine

File: Files/MultilingualResources/Files.fr-FR.xlf [hunk 1]
```diff
@@ -3561,133 +3561,133 @@ Nous utilisons l'App Center pour suivre l'utilisation de l'application, trouver
         </trans-unit>
         <trans-unit id="OpenLogLocation" translate="yes" xml:space="preserve">
           <source>Open log location</source>
-          <target state="new">Open log location</target>
+          <target state="new">Ouvrir l'emplacement des fichiers journaux</target>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.fr-FR.xlf:3564]: Can you update the state to `translated`?

**Review by yaichenbaum — CHANGES_REQUESTED** [review]

File: Files/MultilingualResources/Files.fr-FR.xlf (hunk 1 above)
Comment (Gurgarath) [reply Files/MultilingualResources/Files.fr-FR.xlf]: Done!

**Review by yaichenbaum — COMMENTED** [review]
LGTM

**Review by yaichenbaum — APPROVED** [review]

**Comment by yaichenbaum** [comment]
@Gurgarath can you take a look at #7625?

**Pull request merged**